src/renderer/views/CreatorStudio.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { Col, Container, Row, Button } from 'react-bootstrap';
+
+class CreatorStudio extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {};
+    }
+    componentDidMount() {
+        const webview = document.getElementById('foo')
+        const indicator = document.querySelector('.indicator')
```

Comment:
Could react refs be used instead of default html element selection. See react refs documentation [here](https://reactjs.org/docs/refs-and-the-dom.html)